app/src/ereferrals/java/org/eyeseetea/malariacare/domain/usecase/PullOrganisationCredentialsUseCase.java

```diff
@@ -0,0 +1,114 @@
+package org.eyeseetea.malariacare.domain.usecase;
+
+import org.eyeseetea.malariacare.data.sync.importer.PullOrganisationCredentialsController;
+import org.eyeseetea.malariacare.domain.boundary.executors.IAsyncExecutor;
+import org.eyeseetea.malariacare.domain.boundary.executors.IMainExecutor;
+import org.eyeseetea.malariacare.domain.exception.InvalidCredentialsException;
+import org.eyeseetea.malariacare.domain.exception.NetworkException;
+import org.eyeseetea.malariacare.domain.exception.PullConversionException;
+
+public class PullOrganisationCredentialsUseCase implements UseCase {
```

Comment:
This really is not a domain use case, use case is only login but in erreferrals has two steps but this steps should be encapsulated in LoginUseCase